vllm/engine/metrics.py

```diff
@@ -199,6 +199,33 @@ def __init__(self, labelnames: List[str], vllm_config: VllmConfig):
             documentation="Count of successfully processed requests.",
             labelnames=labelnames + [Metrics.labelname_finish_reason])
 
+        # FLOP counting stats
+        self.counter_total_flops = self._counter_cls(
```

Comment:
It may not make sense to introduce a static flops counter in the engine to get the realtime metrics. Instead, if we really want to monitor the flops number at real time, we'd better try the hardware counter instead.